Dockerfile

```diff
@@ -4,8 +4,10 @@ RUN apt-get update && apt-get -uy upgrade
 RUN apt-get -y install ca-certificates && update-ca-certificates
 
 FROM scratch
-
+# Allow Glog to log to disk
+COPY --from=0 /tmp /tmp
```

Comment:
Are the contents of /tmp really needed? This feels like it could be simplified.

```suggestion
RUN mkdir -p /tmp
```

Reply:
Yeah, that is probably the right solution, good catch, I'll fix it to that.